internal/pkg/encoding/json/schema/schema_test.go

```diff
@@ -281,16 +269,8 @@ func TestValidateObjects_InvalidSchema_Warning2(t *testing.T) {
 }
 `)
 	expectedLogs := `
-WARN  config JSON schema of the component "foo.bar" is invalid, please contact support:
-- anyOf failed:
-  - doesn't validate with "/definitions/simpleTypes":
-    - "properties.foo.type" is invalid: value must be one of "array", "boolean", "integer", "null", "number", "object", "string"
-  - "properties.foo.type" is invalid: expected array, but got string
-WARN  config row JSON schema of the component "foo.bar" is invalid, please contact support:
-- anyOf failed:
-  - doesn't validate with "/definitions/simpleTypes":
-    - "properties.foo.type" is invalid: value must be one of "array", "boolean", "integer", "null", "number", "object", "string"
-  - "properties.foo.type" is invalid: expected array, but got string
+{"level":"warn","message":"config JSON schema of the component \"foo.bar\" is invalid, please contact support:\n- anyOf failed:\n  - doesn't validate with \"/definitions/simpleTypes\":\n    - \"properties.foo.type\" is invalid: value must be one of \"array\", \"boolean\", \"integer\", \"null\", \"number\", \"object\", \"string\"\n  - \"properties.foo.type\" is invalid: expected array, but got string"}
+{"level":"warn","message":"config row JSON schema of the component \"foo.bar\" is invalid, please contact support:\n- anyOf failed:\n  - doesn't validate with \"/definitions/simpleTypes\":\n    - \"properties.foo.type\" is invalid: value must be one of \"array\", \"boolean\", \"integer\", \"null\", \"number\", \"object\", \"string\"\n  - \"properties.foo.type\" is invalid: expected array, but got string"}
```

Comment:
for discussion: in most cases in this PR it works beautifully, ... but in these CLI related things it is quite difficult to read.
Would it be a problem to add an `AllMessagesTxt()` method that would return only messages? To simplify these assertions for the CLI? so there will be no change in the format here.